masonry/src/widget/button.rs

```diff
@@ -104,15 +104,15 @@ impl Widget for Button {
             match event.action {
                 accesskit::Action::Default => {
                     ctx.submit_action(Action::ButtonPressed(PointerButton::Primary));
-                    ctx.request_paint();
+                    ctx.request_paint_only();
```

Comment:
Again, here. How would you expect this paint to be different from the previous one?